travel-agency-api/src/main/java/cz/muni/fi/pa165/bluebat/dto/ReservationDTO.java

```diff
@@ -21,9 +23,25 @@
     @NotNull
     private TripDTO tripDTO;
 
-    @NotNull
-    private PriceDTO priceDTO;
-
-    @NotNull
-    private Set<ExcursionDTO> excursionsDTO;
+    private Set<ExcursionDTO> excursionDTOs = new HashSet<>();
```

Comment:
Tady to taky mohly být je IDčka trip a excursions

Reply:
No ja nad tim premyslel a prislo mi to lepsi takhle, protoze realne by pak ta rezervace nemela nic jineho nez IDcka, ktera stejne nema smysl na frontendu ukazovat, takze by se pred zobrazenim stejne muselo vsechno povytahovat. Je mozne, ze nekde by se nemuselo  vytahovat vsechno (napr. kdyz by si customer chtel zobrazovat svoje rezervace, tak nema smysl tahat u nich customera), ale v takovem pripade mi prijde lepsi pak kdyztak vytvorit dalsi DTO model.

Reply:
Jo to dává smysl :)